src/client/pages/Kiosk/KioskLayout/SidePanel/SidePanel.tsx

```diff
@@ -1,8 +1,45 @@
 import './SidePanel.scss'
-import React from 'react'
+import React, { useTransition } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { Routes } from 'meta/routes'
+
+import Icon from 'client/components/Icon'
 
 const SidePanel: React.FC = () => {
-  return <div className="kiosk-side-panel">side panel</div>
+  const navigate = useNavigate()
+  const [, startTransition] = useTransition()
+
+  return (
+    <div className="kiosk-side-panel">
+      <div className="kiosk-side-panel__header">
+        <img alt="FAO" src="/img/fao/FAOen_3_lines.svg" />
+        <button
+          className="kiosk-side-panel__home-button"
+          onClick={() => startTransition(() => navigate(Routes.Kiosk.path.absolute))}
+          type="button"
+        >
+          <Icon name="home-circle" />
+        </button>
+      </div>
+      <div className="kiosk-side-panel__quiz">
```

Comment:
If we want to hide the quiz for the first deployment we can simply comment it out and the side panel would look like this:
![image](https://github.com/user-attachments/assets/509bb248-abad-4867-ac8b-d3e8bf77feb4)